pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/PersistentTopics.java

```diff
@@ -155,6 +156,46 @@
     }
 
     @GET
+    @Path("/{property}/{cluster}/{namespace}/partitioned")
+    @ApiOperation(value = "Get the list of partitioned topics under a namespace.", response = String.class, responseContainer = "List")
+    @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
+            @ApiResponse(code = 404, message = "Namespace doesn't exist") })
+    public List<String> getPartitionedTopicList(@PathParam("property") String property, @PathParam("cluster") String cluster,
+            @PathParam("namespace") String namespace) {
+        validateAdminAccessOnProperty(property);
+
+        // Validate that namespace exists, throws 404 if it doesn't exist
+        try {
+            policiesCache().get(path("policies", property, cluster, namespace));
+        } catch (KeeperException.NoNodeException e) {
+            log.warn("[{}] Failed to get partitioned topic list {}/{}/{}: Namespace does not exist", clientAppId(), property,
+                    cluster, namespace);
+            throw new RestException(Status.NOT_FOUND, "Namespace does not exist");
+        } catch (Exception e) {
+            log.error("[{}] Failed to get partitioned topic list for namespace {}/{}/{}", clientAppId(), property, cluster, namespace, e);
+            throw new RestException(e);
+        }
+
+        List<String> partitionedTopics = Lists.newArrayList();
+
+        try {
+            String path = path(PARTITIONED_TOPIC_PATH_ZNODE, property, cluster, namespace, domain());
+            List<String> destinations = globalZk().getChildren(path, false);
+            partitionedTopics = destinations.stream().map(s -> String.format("persistent://%s/%s/%s/%s", property, cluster, namespace, decode(s))).collect(
+                    Collectors.toList());
+        } catch (KeeperException.NoNodeException e) {
+            // NoNode means there are no partitioned topics in this domain for this namespace
+        } catch (Exception e) {
+            log.error("[{}] Failed to get partitioned topic list for namespace {}/{}/{}", clientAppId(), property, cluster,
+                    namespace, e);
+            throw new RestException(e);
+        }
+
+        partitionedTopics.sort(null);
```

Comment:
@rdhabalia 
As mentioned above, because I implemented this api under persistent similar like getList which sort, the topic list returned by this api is also sorted.